user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+.. _user-guide-getting-started-company-structure:
+
+Company Structure and Organization Selector
+===========================================
+
+Understanding the Company Structure
+-----------------------------------
+
+Each enterprise is structured in a meaningful way. Big enterprises usually have several branches/offices, each of which 
+may be divided into several major divisions comprised of departments that may be comprised of several functional groups, 
+etc. Even small companies organize their workforce into teams.
+
+For example, a telecommunication service provider might have several subsidiaries, e.g. in each of the states. The 
+Californian subsidiary may have two offices - Western and Eastern. In each of the offices there are unites working at 
+different directions:
+
+
+* CaliforniaTelecom
+
+  * Western California
+ 
+    * Western California Stationary Phones
+  
+    * Western California Mobiles
+
+    * Western California TV
+
+    * Western California Internet
+   
+  * Eastern California
+ 
+    * Eastern California Stationary Phones
+   
+    * Eastern California Mobiles
+
+    * Eastern California TV
+
+    * Eastern California Internet
+
+Each unit in its turn may be divided in teams, for example:
+ 
+- Eastern California Internet Office Employees
+- Eastern California Internet Field Workers
+- Eastern California Internet Freelance Employees
+
+
+The corporate structure is essential, as it defines the scope of the user’s tasks and and available resources. As a 
+business application, OroCRM provides tools to reflect the corporate structure normally.
```

Comment:
Edit: Corporate structure is essential, as it defines the scope of the user’s tasks and available resources. For this reason, OroCRM comes with tools that will reflect the real-world corporate structure of businesses.